charts/grafana/values.yaml

```diff
@@ -1248,6 +1248,18 @@ imageRenderer:
   ##
   # schedulerName: "default-scheduler"
 
+  # Extra configmaps to mount in image-renderer pods
+  extraConfigmapMounts: []
+
+  # Extra secrets to mount in image-renderer pods
+  extraSecretMounts: []
+
+  # Extra volumes to mount in image-renderer pods
+  extraVolumeMounts: []
+
+  # Extra volumes for image-renderer pods
+  extraVolumes: []
```

Comment:
I miss one more thing:

Could you please fillout some new extra values in the CI test config that this values are covered by CI?

https://github.com/grafana/helm-charts/blob/main/charts/grafana/ci/with-image-renderer-values.yaml

For extraVolumes / extraVolumeMounts, empty would be sufficient. 

For extraConfigmapMounts / extraSecretMounts, you may have to setup own ConfigMaps / Secrets, which can be done via extraObject

https://github.com/bradleypettit/helm-charts-grafana-renderer-support-volumes/blob/16c6ee86bb95d69488e0e9cdfd4cd27cbbf30b50/charts/grafana/values.yaml#L1332